pytest_splunk_addon/splunk.py

```diff
@@ -61,13 +61,48 @@ def pytest_addoption(parser):
             "http scheme in the host. default is 127.0.0.1"
         ),
     )
+    group.addoption(
+        "--splunk-hec-scheme",
+        action="store",
+        dest="splunk_hec_scheme",
+        default="https",
+        help="Splunk HTTP event collector port. default is https.",
+    )
+    group.addoption(
+        "--splunk-hec-port",
+        action="store",
+        dest="splunk_hec",
+        default="8088",
+        help="Splunk HTTP event collector port. default is 8088.",
+    )
+    group.addoption(
+        "--splunk-hec-token",
+        action="store",
+        dest="splunk_hec_token",
+        default="9b741d03-43e9-4164-908b-e09102327d22",
+        help='Splunk HTTP event collector token. default is "9b741d03-43e9-4164-908b-e09102327d22".',
+    )
     group.addoption(
         "--splunk-port",
```

Comment:
Now that we have a lot of ports in the option, I think we should rename it as "splunkd-port" or "splunk-mgmt-port" for the users to know that it's a management port.